src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -76,9 +76,14 @@ if (!file_exists("$PHPUNIT_DIR/phpunit-$PHPUNIT_VERSION/phpunit") || md5_file(__
     if (5.1 <= $PHPUNIT_VERSION && $PHPUNIT_VERSION < 5.4) {
         passthru("$COMPOSER require --no-update phpunit/phpunit-mock-objects \"~3.1.0\"");
     }
-    passthru("$COMPOSER require --no-update symfony/phpunit-bridge \"*\"");
-    if (file_exists($path)) {
+    if (file_exists($path = $root.'/vendor/symfony/phpunit-bridge')) {
+        passthru("$COMPOSER require --no-update symfony/phpunit-bridge \"*@dev\"");
```

Comment:
Wouldn't this install a dev version even if a stable version is installed in the project?

Reply:
When the "path" repo is used, the package there is at version "dev-master"
since that's the one we really want to install, we have to put this.